src/test/fixtures/profiles/tracks.js

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import * as profileViewSelectors from '../../../reducers/profile-view';
+import * as urlStateReducers from '../../../reducers/url-state';
+import { getProfileFromTextSamples } from './make-profile';
+import type { Profile } from '../../../types/profile';
+import type { State } from '../../../types/reducers';
+
+export function getHumanReadableTracks(state: State): string[] {
+  const threads = profileViewSelectors.getThreads(state);
+  const globalTracks = profileViewSelectors.getGlobalTracks(state);
+  const hiddenGlobalTracks = urlStateReducers.getHiddenGlobalTracks(state);
+  const selectedThreadIndex = urlStateReducers.getSelectedThreadIndex(state);
+  const text: string[] = [];
+  for (const globalTrackIndex of urlStateReducers.getGlobalTrackOrder(state)) {
+    const globalTrack = globalTracks[globalTrackIndex];
+    const globalHiddenText = hiddenGlobalTracks.has(globalTrackIndex)
+      ? 'hide'
+      : 'show';
+    if (
+      globalTrack.type === 'process' &&
+      globalTrack.mainThreadIndex !== null
+    ) {
+      const selected =
+        globalTrack.mainThreadIndex === selectedThreadIndex ? ' SELECTED' : '';
+      const thread = threads[globalTrack.mainThreadIndex];
+      text.push(
+        `${globalHiddenText} [thread ${thread.name} ${
+          thread.processType
+        }]${selected}`
+      );
+    } else {
+      text.push(`${globalHiddenText} [${globalTrack.type}]`);
+    }
+
+    if (globalTrack.type === 'process') {
+      const trackOrder = urlStateReducers.getLocalTrackOrder(
+        state,
+        globalTrack.pid
+      );
+      const tracks = profileViewSelectors.getLocalTracks(
+        state,
+        globalTrack.pid
+      );
+
+      for (const trackIndex of trackOrder) {
+        const track = tracks[trackIndex];
+        const thread = threads[track.threadIndex];
+        const hiddenTracks = urlStateReducers.getHiddenLocalTracks(
+          state,
+          globalTrack.pid
+        );
+        const hiddenText = hiddenTracks.has(trackIndex) ? 'hide' : 'show';
```

Comment:
I chose `hide` and `show` since they are equal width and very short, which makes for easier to read tests. I originally tried some unicode characters, but this was terse and obvious. So my preference would be to have the slightly funky verb tense.

Reply:
oops I meant to remove this comment before sending the review :)